pyproject.toml

```diff
@@ -21,7 +21,7 @@ classifiers = [
     "Intended Audience :: Science/Research",
     "Operating System :: OS Independent",
 ]
-requires-python = ">=3.10,<=3.11"
+requires-python = ">=3.10,<=3.12"
```

Comment:
keep `">=3.10,<3.12"`

On 3.12 quippy ase does not work

Reply:
Oh, right!